native/src/main/java/io/ballerina/stdlib/mi/OMElementConverter.java

```diff
@@ -77,12 +77,8 @@ private static QName getQNameOMElement(OMElement omElement) {
 
     private static QName getQNameOMAttribute(OMAttribute omAttribute) {
         // If prefix is not provided test case will fail
-        if (omAttribute.getPrefix() == null) {
-            return new QName(omAttribute.getNamespaceURI(), omAttribute.getLocalName());
-        } else {
-            return new QName(omAttribute.getNamespaceURI(), omAttribute.getLocalName(), omAttribute.getPrefix());
-        }
-
+        QName qName = omAttribute.getQName();
+        return new QName(qName.getNamespaceURI(), qName.getLocalPart(), qName.getPrefix());
```

Comment:
Why we can't use above qname as the return value?

Reply:
Fixed